metadata-db-app/src/cmr/metadata_db/services/concept_service.clj

```diff
@@ -399,7 +331,7 @@
         {:message {:warnings [(format "VARIABLE association [%s] is already deleted." concept-id)]}}
         (let [concept {:concept-type :variable-associationn
                        :concept-id concept-id
-                       :user-id (context->user-id mdb-context)
+                       :user-id (context->user-id mdb-context "Associations cannot be modified without a valid user token.")
```

Comment:
I assume this is an error message? Should this not be defined in a messages.clj file?

Reply:
It is.  I have put it into assoc-messages namespace.